target/linux/ramips/dts/ApexeraWiBot.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-wibot", "ralink,mt7620a-soc";
+	model = "Apexera WiBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "wibot:blue:wan";
+			gpios = <&gpio1 5 1>;
+		};
+
+		wifi {
+			label = "wibot:blue:wifi";
+			gpios = <&gpio1 4 1>;
+		};
+
+		nand {
+			label = "wibot:blue:nand";
+			gpios = <&gpio1 3 1>;
+		};
+
+		boot {
+			label = "wibot:blue:boot";
+			gpios = <&gpio1 11 1>;
+			default-state = "on";
+		};
+
+		reset {
+			label = "wibot:blue:reset";
+			gpios = <&gpio2 23 1>;
+		};
+
+		ctrl3g {
+			label = "wibot:blue:ctrl3g";
+			gpios = <&gpio2 21 1>;
+		};
+
+		usb0 {
+			label = "wibot:blue:usb0";
+			gpios = <&gpio2 20 1>;
+			default-state = "on";
+		};
+
+		usb1 {
+			label = "wibot:blue:usb1";
+			gpios = <&gpio2 22 1>;
+		};
+        };
+};
+
+&gpio0 {
+	status = "okay";			
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&uart {
+	status = "okay";
+};
+
+&i2c {
+	status = "okay";
+
+	pcf8563: rtc@51 {
+		status = "okay";
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "w25q128";
+		reg = <0 0>;
+		linux,modalias = "m25p80", "w25q128";
+		spi-max-frequency = <10000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+	    	partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+		        };
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				label = "firmware";
```

Comment:
please add the compatible `compatible = "denx,uimage";` check the other dts files for reference.